### PR TITLE
Adds notes for 4.11.5 release

### PR DESCRIPTION
**Pull request by opayne1**
Adds notes for tomorrow's 4.11.5 release

Issue:
[OSDOCS-4143](https://issues.redhat.com/browse/OSDOCS-4143)

Link to docs preview:
https://50540--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-5

Additional information:
Links will not work yet!

### CODE REVIEW

**Comment by opayne1** [comment]
closing and reopening to get travis to build

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -2608,3 +2608,32 @@ $ oc adm release info 4.11.4 --pullspecs
...
+
+[id="ocp-4-11-5-known-issues"]
+==== Known issues
+
+* Sharding the default Ingress Controller breaks the {product-title} factory routes such as `canary`, `oauth`, and `console`. As a workaround, you can manually add matching labels and expressions to the routes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024946[*BZ#2024946*])
+
+[id="ocp-4-11-5-bug-fixes"]
+==== Bug fixes
+
+* Previously, after a `routeSelector` update, the Ingress Controller cleared the route status before the router deployment finished rolling out, resulting in the status being repopulated incorrectly. With this update, the route status is cleared with a `routeSelector` update. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110528[*BZ#2110528*])
```
Comment (JoeAldinger) [inline release_notes/ocp-4-11-release-notes.adoc:2634]:
```suggestion
* Previously, an update of the `routeSelector` cleared the route status of the Ingress Controller before the router deployment finished rolling out, which resulted in the status being repopulated incorrectly. With this update, the route status is cleared with a `routeSelector` update. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110528[*BZ#2110528*])
```

Comment (michaelryanpeter) [reply release_notes/ocp-4-11-release-notes.adoc:2634]:
```suggestion
*  Previously, update of the `routeSelector` cleared the route status of the Ingress Controller prior to the router deployment. As a result, the route status repopulated incorrectly. With this update, the route status is cleared with a `routeSelector` update. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110528[*BZ#2110528*])
```
Just an additional idea, not a blocker. I sometimes get lost in bug fixes, due to their compactness and chain of events. As a result, I try to make each sentence as short as I can.

Comment (opayne1) [reply release_notes/ocp-4-11-release-notes.adoc]: Thanks for the suggestions! I have updated.

**Pull request merged**